### DISCUSSION TITLE
Intermittent sound

### DISCUSSION THREAD

**Discussion by robin28** [jamulussoftware/jamulus#1460] [General]
I get intermittent sound.  Whatever is going on in Jamulus, I hear as a sequence of pulses, ¼ second of distorted sound, ¾ second quiet.  I can tell that they are speaking or singing but understand nothing.  New Behringer UMC22 (on a DELL laptop) works fine with Audacity.  Anyone heard this before?

**Comment by dcorson-ticino-com** [comment]
Are you using a cabled Internet connection?  That sounds like one of the many incarnations of problems caused by wifi.

**Comment by robin28** [comment]
This machine seems to prioritise ethernet over WiFi but I turn WiFi off anyway.

**Comment by ann0see** [comment]
What's your buffer size? I think your device doesn't work very well with ASIO4ALL and low buffer sizes.

**Comment by dakhubgit** [comment]
ASIO4ALL produces garbage on these cards with the buffer sizes Jamulus uses for some reason.  You need to look for the old Behringer ASIO driver that they distributed a few years ago for that card.  Maybe on archive.org, maybe somewhere else.
